code/game/objects/items/storage/briefcase.dm

```diff
@@ -35,6 +35,24 @@
 	new /obj/item/stamp/law(src)
 	..()
 
+/obj/item/storage/briefcase/suicide_act(mob/user)
+	var/list/papers_found = list()
+	var/turf/T = get_turf(src)
+	for(var/atom/A in contents)
+		if(istype(A, /obj/item/paper) || istype(A, /obj/item/paperplane))
+			papers_found += A
+	if(!LAZYLEN(papers_found) || !T)
```

Comment:
Don't use `LAZYLEN`, just use `papers_found.len`--it's not a lazy list.

Also, you can check if the turf is valid before running the contents loop.